make_installer.iss

```diff
@@ -38,7 +39,8 @@ Name: "english"; MessagesFile: "compiler:Default.isl"
 Name: "desktopicon"; Description: "{cm:CreateDesktopIcon}"; GroupDescription: "{cm:AdditionalIcons}"; Flags: unchecked
 
 [Files]
```

Comment:
Understand that this will cause some issues due to it being my file structure but we should be the only people to touch this so just change it as you need.